### DISCUSSION TITLE
[feature request] Powder keg blast mask.

### DISCUSSION THREAD

**Discussion by smashbro596** [HarbourMasters/2ship2harkinian#569] [Ideas]
ok so i was playing around with the blast mask and thought to myself "what if i made this into a powder keg?"
so to start off, pressing b will start the powder keg countdown. you run faster (slightly faster than the bunny hood) and you hear yourself ticking. when the time runs out, or an enemy touches you, or if you press the b button again, you unleash a powder keg explosion, with all its properties (namely blowing up the rock blocking romani ranch).  if infinite health isn't turned on, this does three hearts worth of damage to you. it also takes twice as long to recharge if instant blast mask cooldown isn't active.

**Comment by Caladius** [comment]
https://github.com/HarbourMasters/2ship2harkinian/pull/642
